### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by filip-michalsky**
Documentation for jurity library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,36 @@
 # Jurity: Fairness & Evaluation Library
 
-Jurity is a research library that provides fairness metrics, recommender system evaluations and bias mitigation techniques. The library adheres to PEP-8 standards and is tested heavily. 
+Jurity is a research library that provides classification metrics, fairness metrics, recommender system evaluations, and bias mitigation techniques. The library adheres to PEP-8 standards and is tested heavily.
```
Comment (bkleyn) [inline README.md:3]: Not sure if classification needs to be added here. Fairness and reco should really be the focus. If we add it, it should be after reco system evals in the sentence.

File: docs/_sources/about.rst.txt [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+Binary Recommender Metrics
+--------------------------
+Binary recommender metrics directly measure the click interaction.
+
+CTR: Click-through Rate
+^^^^^^^^^^^^^^^^^^^^^^^
+
+CTR measures the accuracy of the recommendations over the subset of user-item pairs that appear in both actual ratings and recommendations.
+
+Let :math:`M` denote the set of user-item pairs that appear in both actual ratings and recommendations, and :math:`C(M_i)` be an indicator function that procudes :math:`1` if the user clicked on the item, and :math:`0` if they didn't.
```
Comment (bkleyn) [inline docs/_sources/about.rst.txt:104]: Typo: procudes > produces

File: docs/_sources/about.rst.txt [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+Equalized Odds
+^^^^^^^^^^^^^^
+
+Equalized odds is a bias mitigation technique where subset of decisions of a binary classifier is flipped at uniform random in each of two groups to achieve equality of TPR and FPR across the two groups as proposed in [2]_. This subset rate in each group is learned via constrained optimization.
+
+References:
+            .. [2] Moritz Hardt, Eric Price, and Nathan Srebro. 2016. Equality of opportunity in supervised learning. In Proceedings of the 30th International Conference on Neural Information Processing Systems (NIPS'16).
+
+About Recommenders Metrics
```
Comment (bkleyn) [inline docs/_sources/about.rst.txt:85]: Do we want Fairness/Reco metrics on the same page? They are in different sections in TOC.

File: README.md [hunk 4]
```diff
@@ -104,6 +112,25 @@ print("Recall@K:", recall_k.get_score(actual, predicted))
 print("MAP@K:", map_k.get_score(actual, predicted))
 ```
 
+## Quick Start: Classifier Evaluation
```
Comment (bkleyn) [inline README.md:115]: Classifier > Classification to be consistent with Quick Start in docs

File: docs/_sources/api.rst.txt [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    
+Binary Bias Mitigation
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: jurity.mitigation.BinaryMitigation
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Binary Recommenders Metrics
```
Comment (bkleyn) [inline docs/_sources/api.rst.txt:43]: Some of the doc string lengths go beyond 120 and are not being wrapped nicely.

File: docs/_sources/api.rst.txt [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+Jurity Public API
+====================
+
+.. automodule:: jurity
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Binary Classification Metrics
```
Comment (bkleyn) [inline docs/_sources/api.rst.txt:11]: Returns and Return Type are not consistently populated. Think it's the same for some of our other libraries so not a big issue, but wanted to make a note of this.

File: docs/_sources/about.rst.txt [hunk 7]
```diff
@@ -0,0 +1,150 @@
+.. _about:
+
+About Algorithmic Fairness
+==========================
+
+Below, see the mathematical definition for each of the fairness metrics in the library.
+
+Average odds
```
Comment (bkleyn) [inline docs/_sources/about.rst.txt:8]: Consistent capitalization: "Average Odds"

File: docs/_sources/about.rst.txt [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+==========================
+
+Jurity offers various standardized metrics for measuring the recommendation performance.
+While the recommendation systems community agrees on these metrics, the implementations can be different, especially when it comes to edge-cases.
+
+For the definitions below, :math:`A` is the set of actual ratings for users and :math:`P` is the set of predictions / recommendations.
+Each :math:`A_i` and :math:`P_i` represents the list of actual items and list of recommended items, respectively, for a user :math:`i`.
+
+
+Binary Recommender Metrics
```
Comment (bkleyn) [inline docs/_sources/about.rst.txt:95]: Not sure if docs haven't been updated, but something looks off with heading structure on page.

image

File: README.md (hunk 1 above)
Comment (skadio) [reply README.md:3]: +1

File: docs/_sources/about.rst.txt (hunk 3 above)
Comment (skadio) [reply docs/_sources/about.rst.txt:85]: would be nice to keep that separate, since they are logically different parts of the library

**Comment by skadio** [comment]
good to go from my end, with the minor revisions above

File: docs/_sources/quick.rst.txt [hunk 9]
```diff
@@ -0,0 +1,102 @@
+.. _quick:
+
+Quick Start 
+===========
+
+Calculate Fairness Metrics:
```
Comment (dorukkilitcioglu) [inline docs/_sources/quick.rst.txt:6]: We mentioned adding anchors to the quick start examples before, we should add them.

File: README.md (hunk 1 above)
Comment (filip-michalsky) [reply README.md:3]: We discussed with @skadio that we will just mention we have the classification metrics as well

File: docs/_sources/api.rst.txt (hunk 5 above)
Comment (filip-michalsky) [reply docs/_sources/api.rst.txt:43]: @dorukkilitcioglu some of the print statements are still not wrapped correctly... This *may* be an issue with me and doruk using a different sphinx version.

File: docs/_sources/about.rst.txt (hunk 8 above)
Comment (filip-michalsky) [reply docs/_sources/about.rst.txt:95]: this should be all set

File: docs/_sources/api.rst.txt (hunk 5 above)
Comment (dorukkilitcioglu) [reply docs/_sources/api.rst.txt:43]: Updated this now, the code changes didn't show up in the documentation, so I re-compiled and pushed.

**Comment by filip-michalsky** [comment]
All comments except the `typing` are addressed. The `typing` comment is more about the library itself than the documentation and should be addressed in a subsequent PR.

@bkleyn good to merge from my side if you are good.

**Pull request merged**